src/common/kits/CaseManager.ts

```
@@ -0,0 +1,64 @@
+      colorText: "#FFA800",
+      text: "Final Review",
+    },
+    [CaseStatusEnum.DEFAULT]: {
+      colorBackground: "#F5F5F5",
+      colorText: "#999999",
+      text: "",
+    },
+  };
+
+  static getInstance(): CaseManager {
+    if (!this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)